.eslintrc.json

```diff
@@ -40,15 +40,19 @@
             "depConstraints": [
               {
                 "sourceTag": "scope:core",
-                "onlyDependOnLibsWithTags": ["scope:shared"]
+                "onlyDependOnLibsWithTags": ["scope:shared", "scope:utils"]
               },
               {
                 "sourceTag": "scope:shared",
-                "onlyDependOnLibsWithTags": ["scope:shared"]
+                "onlyDependOnLibsWithTags": ["scope:shared", "scope:utils"]
               },
               {
                 "sourceTag": "scope:integration",
-                "onlyDependOnLibsWithTags": ["scope:shared"]
+                "onlyDependOnLibsWithTags": ["scope:shared", "scope:utils"]
+              },
+              {
+                "sourceTag": "scope:utils",
+                "onlyDependOnLibsWithTags": ["scope:utils"]
```

Comment:
Is there a reason for a new scope? Could we just use `shared`?

Reply:
Ahh sorry, I already changed it to `shared` from Peng's feedback yesterday. Just haven't pushed the change.